composer.json

```diff
@@ -153,7 +153,8 @@
         "enable-patching": true,
         "patches": {
             "symfony/dependency-injection": [
-                "https://raw.githubusercontent.com/rectorphp/vendor-patches/main/patches/symfony-dependency-injection.patch"
+                "https://raw.githubusercontent.com/rectorphp/vendor-patches/main/patches/symfony-dependency-injection.patch",
+                "https://raw.githubusercontent.com/rectorphp/vendor-patches/main/patches/symfony-php-config-loader.patch"
             ]
```

Comment:
After this PR is merged, line needs to update in each Rector split repository's `composer.json`, so we have the same patched Symfony everywhere